SQL/database_schema_prefixed.sql

```diff
@@ -148,24 +227,13 @@ CREATE TABLE IF NOT EXISTS `rp_round` (
 CREATE TABLE IF NOT EXISTS `rp_connection_log` (
   `id` INT(11) NOT NULL AUTO_INCREMENT,
   `datetime` datetime NOT NULL,
-  `serverip` varchar(16) NOT NULL,
+  `serverip` varchar(45) NOT NULL,
   `ckey` varchar(32) NOT NULL,
-  `ip` varchar(16) NOT NULL,
+  `ip` varchar(45) NOT NULL,
```

Comment:
quick note for sql people:

this is objectively shit because it's kind of stupid to store this stuff in string and not binary

but today isn't the day i refactor banning and admin system so, rip lmao